codespell_lib/tests/test_basic.py

```diff
@@ -49,8 +49,7 @@ def main(
             assert code == 0
         if std:
             return (code, stdout, stderr)
-        else:
-            return code
+        return code
```

Comment:
I know pyupgrade (or is it refurb?) suggest this, but I find this less readable. I am not necessarily against it, but which tools suggests this change?